tests/meta/store/snapshots/snap_test_cg_meta_store_base.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
```

Reply:
removed, again, in https://github.com/Clinical-Genomics/cg/pull/621/commits/eaf0014ca3a9917621236ace8ea80ee67c3f8d5d